### PR TITLE
Adding a test for the Theme Categories homepage shelf

### PR DESCRIPTION
**Pull request by AlexandraMoga**
Adding a test for the Theme Categories homepage shelf which verifies the content, clicks through each category and verifies that the correct search result page is opened. 

The shelf in question:
image

### CODE REVIEW

File: test_home.py [hunk 1]
```diff
@@ -54,6 +54,30 @@ def test_more_dropdown_navigates_correctly(base_url, selenium, i, page_url):
...
+        'Film',
+        'Scenery',
+        'Music',
+        'Seasonal',
+    ])
+)
+@pytest.mark.nondestructive
+@pytest.mark.desktop_only
+def test_theme_categories_shelf(base_url, selenium, count, category):
+    page = Home(selenium, base_url).open()
```
Comment (jrbenny35) [inline test_home.py:71]: Could we add a comment here with a brief explanation?

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Sweeet test!

File: test_home.py (hunk 1 above)
Comment (AlexandraMoga) [reply test_home.py:71]: I've added two comments to make it more clear what this test is about

**Review by jrbenny35 — APPROVED** [review]
Thanks Lexa!

**Pull request merged**